library/std/src/io/impls.rs

```diff
@@ -565,6 +565,11 @@ impl<A: Allocator> Write for VecDeque<u8, A> {
         Ok(len)
     }
 
+    fn write_all_vectored(&mut self, bufs: &mut [IoSlice<'_>]) -> io::Result<()> {
```

Comment:
Is there a reason this is not `#[inline]`?

Reply:
I think that the policy is to avoid `#[inline]` by default, but I can add it for consistency with other methods.

Reply:
Almost all the other methods in this file have `#[inline]`, so I'd think these should have it too. The only two exceptions seem to have been mistakes and I'm adding them in a soon-to-appear PR.